fs/read_file_str.ts

```diff
@@ -0,0 +1,29 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * Read file synchronously and output it as a string.
+ *
+ * @param filename File to read
+ * @param [encoding="utf-8"] encoding Encoding of the file
+ */
+export function readFileStrSync(
+  filename: string,
+  encoding: string = "utf-8"
+): string {
+  const decoder = new TextDecoder(encoding);
+  return decoder.decode(Deno.readFileSync(filename));
+}
+
+/**
+ * Read file and output it as a string.
+ *
+ * @param filename File to read
+ * @param [encoding="utf-8"] Encoding of the file
+ */
+export async function readFileStr(
```

Comment:
how about

```suggestion
export async function readFile(
```

Reply:
May be confusing with the Deno.readFile no?

Reply:
@zekth
`fs` module is used to add some quick and easy methods

Returning a string is the most common method for developer

In most cases, `deno.readFile` returns the `Reader` is not commonly used

So I don’t think these conflicts.

Reply:
Totally agree with you. First it was named like this to be in `Deno` and not in `fs-extra` as you can see in the ref.